README.md

```diff
@@ -41,7 +41,7 @@ Click [here](https://rotki.readthedocs.io/en/latest/changelog.html) for the late
 
 ## Contribute
 
-rotki is an opensource project and as such any and all help is really appreciated.
+rotki is an open source project and as such any and all help is really appreciated.
```

Comment:
opensource can also be one word

Reply:
open source is written 3 times on the same page, it's also for consistency

Reply:
In that case that is fine :)